From e3688be47d2b72f130f90dafd24b5f5acc4684ca Mon Sep 17 00:00:00 2001
From: Jeevan Shriram <jshriram@codeaurora.org>
Date: Thu, 5 Oct 2017 13:30:06 -0700
Subject: QcomModulePkg: Reset the Partition name-check variable

Reset the partition name check variable for every iteration
to ensure that partition name is properly terminated in the
META image.

Change-Id: Ic55c5ef24a1e10fe472f849a4e0670b5b0bc0d3b
---
 QcomModulePkg/Library/FastbootLib/FastbootCmds.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/QcomModulePkg/Library/FastbootLib/FastbootCmds.c b/QcomModulePkg/Library/FastbootLib/FastbootCmds.c
index 187d716..f60cf2c 100644
--- a/QcomModulePkg/Library/FastbootLib/FastbootCmds.c
+++ b/QcomModulePkg/Library/FastbootLib/FastbootCmds.c
@@ -902,6 +902,8 @@ HandleMetaImgFlash(
 
 	for (i = 0; i < images; i++)
 	{
+            PnameTerminated = FALSE;
+
 		if (img_header_entry[i].ptn_name == NULL || img_header_entry[i].start_offset == 0 || img_header_entry[i].size == 0)
 			break;
 
-- 
cgit v1.1

